packages/curve-ui-kit/src/themes/components.ts

```diff
@@ -47,6 +48,12 @@ export const createComponents = (design: DesignSystem): ThemeOptions['components
       paper: { maxHeight: '100dvh', [basicMuiTheme.breakpoints.down('tablet')]: { margin: 0 } },
     },
   },
+  MuiFormControlLabel: {
+    styleOverrides: {
+      root: { margin: '0' }, // by default there is a negative margin 🤦
```

Comment:
lol what's up with that, I encountered the same thing with toggle buttons, specifically the middle button in a group had a `margin-left: -1px` 🤷

Reply:
Yeah I think they did this for backwards compatibility with some old versions :disappointed: